src/Domain/Aggregate/Pizzeria.php

```diff
@@ -33,12 +39,28 @@ public static function new($name): self
         return $instance;
     }
 
+    public function addOrder(string $customerName, string $pizzaTaste)
+    {
+        $this->recordThat(OrderReceived::fromCustomerPizzeriaAndPizzaTaste(
```

Comment:
we should check that the customerName is not empty (and maybe that the pizzaTaste is a valid one?)